config/database.go

```diff
@@ -2,11 +2,17 @@ package config
 
 import (
 	"fmt"
+	"github.com/jinzhu/gorm"
 	_ "github.com/kelseyhightower/envconfig"
 	"gopkg.in/yaml.v2"
 	"os"
 )
 
+var (
+	DB  *gorm.DB
+	err error
```

Comment:
why error is being initialised as global variable?

Reply:
See line 66. If i keep it local then can't assign to DB variable which is global and err which would be local